doc/changes/DM-40443.removal.rst

```diff
@@ -0,0 +1,9 @@
+Removed deprecated code scheduled to be removed after v27:
```

Comment:
@TallJimbo can you look at the news fragment please?

Reply:
:+1: